apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentReadResourceIntegrationTest.java

```diff
@@ -2631,7 +2631,12 @@ public void escapedChar() {
           .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
           .queryParam("raw", true)
           .when()
-          .get(BASE_PATH + "/{document-id}/a%5C.b", DEFAULT_NAMESPACE, DEFAULT_COLLECTION, id)
+          .get(
+              BASE_PATH + "/{document-id}/{leaf}",
```

Comment:
Earlier code was trying to pre-encode path but it really needs to be left to framework to do with template.
This may have "worked" with conjuction with bug Quarkus 3.2.1 fixed but is incorrect, leading to double-encoding by caller and then mismatching on server side.